src/uct/ib/rc/base/rc_iface.c

```diff
@@ -704,6 +704,8 @@ unsigned uct_rc_iface_qp_cleanup_progress(void *arg)
 
     if (cleanup_ctx->cq_credits > 0) {
         uct_rc_iface_add_cq_credits_dispatch(iface, cleanup_ctx->cq_credits);
+        ucs_arbiter_dispatch(&iface->tx.arbiter, 1, uct_rc_ep_process_pending,
+                             NULL);
```

Comment:
why needed?

Reply:
we removed it from `uct_rc_iface_add_cq_credits_dispatch`

Reply:
can use uct_rc_iface_arbiter_dispatch here?

Reply:
fixed